File: python/S01-elemental-actions/E04-clock-angle.py
def main():
	print("\n\033[35m[========= CLOCK ANGLE =========]\033[0m\n")

	hourValue: int = int(input("Enter the hour: "))
	minuteValue: int = int(input("Enter the minute: "))

	hourAngle: float = hourValue * 30
	minuteAngle: float = minuteValue * 6
	angleDifference: float = minuteAngle / 12

	finalAngle: float = abs(( hourAngle + angleDifference ) - minuteAngle)

	print(f"\nThe final angle is {finalAngle}")


if __name__ == "__main__":
	main()

File: c++/S01-elemental-actions/E04-clock-angle.cpp
#include <iostream>
#include <cmath>

int main(int argc, char *argv[]) {
	std::cout << "\n\e[0;35m[========= CLOCK ANGLE =========]\e[0m\n\n";

	float hourHand, minuteHand;

	std::cout << "Enter the clock hour hand: ";
	std::cin >> hourHand;
	std::cout << "Enter the clock minute hand: ";
	std::cin >> minuteHand;

	float hourAngle, minuteAngle, finalAngle;

	hourAngle = hourHand * 30;
	minuteAngle = minuteHand * 6;
	finalAngle = abs((hourAngle + (minuteAngle / 12)) - minuteAngle);

	std::cout << "The angle between both hand's is " << finalAngle << " degrees." << '\n';

	return 0;
}
